packages/cli/src/services/checkly-config-loader.ts

```diff
@@ -50,6 +50,15 @@ export type ChecklyConfig = {
        */
       testMatch?: string | string[],
     },
+    /**
+     * Multistep checks default configuration properties.
+     */
+    multistepChecks?: CheckConfigDefaults & {
```

Comment:
For the naming convention - It looks like we're already exposing `multiStep` arguments and `MultiStepCheck` as the construct

I think we should make sure to capitalize `Step` everywhere then in order to stay consistent.

https://github.com/checkly/checkly-cli/blob/bfc68daeeacc92cfe427e20305a2a36dc0fac043/packages/cli/src/constructs/check-group.ts#L93

https://github.com/checkly/checkly-cli/blob/bfc68daeeacc92cfe427e20305a2a36dc0fac043/packages/cli/src/constructs/multi-step-check.ts#L28